nautobot/docs/user-guide/platform-functionality/staticgroup.md

```diff
@@ -0,0 +1,36 @@
+
+A given Static Group may contain any number of objects of the appropriate type as members, and an object may belong to any number of Static Groups of the appropriate content-type as a member of each.
+
+When creating a Static Group, you must provide a unique name for the group and define the Content Type of objects that it contains, for example `dcim.device`. You can also optionally provide a description for the group and apply [Tags](tag.md) to it.
+
+Once created, the Content Type for a Static Group may not be changed, as doing so would invalidate any existing members of this group. The name, description, and tags may be updated if desired.
+
+Each association of a given object into a given Static Group is recorded by a [Static Group Association](staticgroupassociation.md) database record.
+
+## REST API
+
+The REST API endpoints under `/api/extras/static-groups/` are used to perform standard CRUD (Create/Retrieve/Update/Delete) operations on Static Group definitions. There is also a special read-only endpoint `/api/extras/static-groups/<group UUID>/members/` that returns a serialized list of the group's member objects; for example, for a Static Group of Prefixes, this endpoint would return a list of the Prefix objects contained in this Static Group.
+
+As in the database itself, in the REST API, assignment of individual objects to an individual Static Group is managed via the endpoints under `/api/extras/static-group-associations/`. Here you can perform CRUD operations to associate objects to an existing Static Group, retrieve the list of associations, and update/delete existing object associations.
+
+## Python API
+
+From a given `StaticGroup` record, the following Python APIs are available:
+
+* `group.members` - returns a `QuerySet` directly representing the objects (for example, a `QuerySet` of `Prefix` records for a Static Group of Prefixes) that are members of this group
+* `group.static_group_associations` - returns a `QuerySet` of `StaticGroupAssociation` records representing the assignment of objects to this group
+* `group.add_members(list_or_queryset)` - associate the given objects to this group as members by creating the appropriate `StaticGroupAssociation` database records
+* `group.remove_members(list_or_queryset)` - disassociate the given objects from this group by deleting any appropriate `StaticGroupAssociation` database records
+
+From a given record of any object type that can be a member of a Static Group, the following Python APIs are available:
+
+* `object.static_groups` - returns a `QuerySet` directly representing the `StaticGroup` records that contain this object as a member
+* `object.associated_static_groups` - returns a `QuerySet` of `StaticGroupAssociation` records representing the assignment of this object to static groups
```

Comment:
`associated_static_groups` implies this is returning static groups. Should this be named `static_group_associations` instead?

Reply:
I was following the same pattern as `object.associated_contacts`, but I agree it's not super clear.

The problem with using `static_group_associations` is that on `StaticGroup` itself that lookup name becomes ambiguous which side of the M2M it represents.